program/src/api/storage.rs

```diff
@@ -79,7 +79,7 @@ mod gear {
     impl Api {
         /// Get `InstrumentedCode` by `code_hash`
         pub async fn code_storage(&self, code_hash: [u8; 32]) -> Result<Option<InstrumentedCode>> {
-            let at = storage().gear_program().code_storage(&CodeId(code_hash));
+            let at = storage().gear_program().code_storage(CodeId(code_hash));
```

Comment:
strange diff, it's on master already

Reply:
Yes, the same commit is on master. removing this commit will require force push, so let's ignore it. It'will be merged 'cause of the same commit hash